config/locales/en.yml

```diff
@@ -40,4 +40,8 @@ en:
         other: ">%{count}yr"
       almost_x_years:
         one:   "~1yr"
-        other: "~%{count}yr"
\ No newline at end of file
+        other: "~%{count}yr"
+  activerecord:
+    attributes:
+      agent:
+          receivers: Targets
```

Comment:
I thought `Targets` would match the context a bit better then `Receivers`, but I would rather call it Recievers then renaming a table and replacing a dozens of `receivers` usages in the code :smile: Actually, since we are using the agent model a lot in migrations, renaming the table could cause problems. When a user is migration from an old version and a migrations tries to call `agent.targets` the migration will not work because the table renaming is not done yet.